The job **`x86_64-gnu-llvm-16`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/7272213186/job/19813969242) [(plain)](https://github.com/rust-lang/rust/commit/fd1bc8c893cd1939e05b056265ea3ce662ac95c4/checks/19813969242/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
GITHUB_ACTION=__run_7
GITHUB_ACTIONS=true
GITHUB_ACTION_REF=
GITHUB_ACTION_REPOSITORY=
GITHUB_ACTOR=rursprung
GITHUB_API_URL=https://api.github.com
GITHUB_BASE_REF=master
GITHUB_ENV=/home/runner/work/_temp/_runner_file_commands/set_env_9f155d24-1bc9-4a36-934e-e5366a4de0f1
GITHUB_EVENT_NAME=pull_request
---
GITHUB_SERVER_URL=https://github.com
GITHUB_SHA=64ada182c65f1d0e7a855e1b2044d63038eb0e0e
GITHUB_STATE=/home/runner/work/_temp/_runner_file_commands/save_state_9f155d24-1bc9-4a36-934e-e5366a4de0f1
GITHUB_STEP_SUMMARY=/home/runner/work/_temp/_runner_file_commands/step_summary_9f155d24-1bc9-4a36-934e-e5366a4de0f1
GITHUB_TRIGGERING_ACTOR=rursprung
GITHUB_WORKFLOW_REF=rust-lang/rust/.github/workflows/ci.yml@refs/pull/119153/merge
GITHUB_WORKFLOW_SHA=64ada182c65f1d0e7a855e1b2044d63038eb0e0e
GITHUB_WORKSPACE=/home/runner/work/rust/rust
GOROOT_1_19_X64=/opt/hostedtoolcache/go/1.19.13/x64
---
Built container sha256:9c3c93a371e5aed5c18185b24f130d95d5140dbd72a9b325e7b6b49e521a4faa
Looks like docker image is the same as before, not uploading
https://ci-caches.rust-lang.org/docker/7ebc15c01a233894034d277c8cce4e949f4e7791f66b4727c8fb6e058a0b8171d6152e1441d677cef0653843ceeee469c097b8699b2bb74249e674f6aa1a8813
sha256:9c3c93a371e5aed5c18185b24f130d95d5140dbd72a9b325e7b6b49e521a4faa
Setting extra environment values for docker:  --env ENABLE_GCC_CODEGEN=1 --env GCC_EXEC_PREFIX=/usr/lib/gcc/
[CI_JOB_NAME=x86_64-gnu-llvm-16]
##[group]Clock drift check
  local time: Wed Dec 20 07:18:59 UTC 2023
  network time: Wed, 20 Dec 2023 07:18:59 GMT
  network time: Wed, 20 Dec 2023 07:18:59 GMT
##[endgroup]
sccache: Starting the server...
##[group]Configure the build
configure: processing command line
configure: 
configure: build.configure-args := ['--build=x86_64-unknown-linux-gnu', '--llvm-root=/usr/lib/llvm-16', '--enable-llvm-link-shared', '--set', 'rust.thin-lto-import-instr-limit=10', '--enable-verbose-configure', '--enable-sccache', '--disable-manage-submodules', '--enable-locked-deps', '--enable-cargo-native-static', '--set', 'rust.codegen-units-std=1', '--set', 'dist.compression-profile=balanced', '--dist-compression-formats=xz', '--disable-dist-src', '--release-channel=nightly', '--enable-debug-assertions', '--enable-overflow-checks', '--enable-llvm-assertions', '--set', 'rust.verify-llvm-ir', '--set', 'rust.codegen-backends=llvm,cranelift,gcc', '--set', 'llvm.static-libstdcpp', '--enable-missing-tools', '--enable-new-symbol-mangling']
configure: target.x86_64-unknown-linux-gnu.llvm-config := /usr/lib/llvm-16/bin/llvm-config
configure: llvm.link-shared     := True
configure: rust.thin-lto-import-instr-limit := 10
configure: rust.codegen-units-std := 1
---
   Compiling addr2line v0.21.0
error[E0546]: missing 'feature'
   --> library/std/src/fs.rs:423:5
    |
423 |     #[stable(since = "CURRENT_RUSTC_VERSION")]

error: associated function has missing stability attribute
   --> library/std/src/fs.rs:424:5
    |
    |
424 | /     pub fn create_new<P: AsRef<Path>>(path: P) -> io::Result<File> {
425 | |         OpenOptions::new().read(true).write(true).create_new(true).open(path.as_ref())
    | |_____^

For more information about this error, try `rustc --explain E0546`.
error: could not compile `std` (lib) due to 2 previous errors
```

</details>

--- a/library/std/src/fs.rs
+++ b/library/std/src/fs.rs
@@ -411,8 +411,6 @@ impl File {
     /// # Examples
     ///
     /// ```no_run
-    /// #![feature(file_create_new)]
-    ///
     /// use std::fs::File;
     /// use std::io::Write;
     ///
@@ -422,7 +420,7 @@ impl File {
     ///     Ok(())
     /// }
     /// ```
-    #[unstable(feature = "file_create_new", issue = "105135")]
+    #[stable(feature = "file_create_new", since = "CURRENT_RUSTC_VERSION")]
     pub fn create_new<P: AsRef<Path>>(path: P) -> io::Result<File> {
         OpenOptions::new().read(true).write(true).create_new(true).open(path.as_ref())
     }

